File: app/src/main/java/com/talktiva/pilot/results/ResultEvents.java
package com.talktiva.pilot.results;

import com.google.gson.annotations.SerializedName;
import com.talktiva.pilot.model.Event;
import com.talktiva.pilot.model.Links;

import java.util.List;

public class ResultEvents {

    @SerializedName("content")
    private List<Event> events;

    @SerializedName("totalElements")
    private int totalElements;

    @SerializedName("totalPages")
    private int totalPages;

    @SerializedName("totalCount")
    private int totalCount;

    @SerializedName("size")
    private int size;

    @SerializedName("currentPage")
    private int currentPage;

    @SerializedName("links")
    private Links links;

    public List<Event> getEvents() {
        return events;
    }

    public int getTotalElements() {
        return totalElements;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public int getTotalCount() {
        return totalCount;
    }

    public int getSize() {
        return size;
    }

    public int getCurrentPage() {
        return currentPage;
    }

    public Links getLinks() {
        return links;
    }
}

File: app/src/main/java/com/talktiva/pilot/results/ResultEvents.kt
package com.talktiva.pilot.results

import com.google.gson.annotations.SerializedName
import com.talktiva.pilot.model.Event
import com.talktiva.pilot.model.Links
import java.io.Serializable

class ResultEvents : Serializable {

    @SerializedName("content")
    var events: List<Event>? = null

    @SerializedName("totalElements")
    var totalElements: Int? = null

    @SerializedName("totalPages")
    var totalPages: Int? = null

    @SerializedName("totalCount")
    var totalCount: Int? = null

    @SerializedName("size")
    var size: Int? = null

    @SerializedName("currentPage")
    var currentPage: Int? = null

    @SerializedName("links")
    var links: Links? = null
}
